src/gpio.rs

```diff
@@ -183,6 +183,12 @@ pub struct Gpiox {
     index: u8,
 }
 
+// NOTE(unsafe) we only use atomic operations on the gpio registers
```

Comment:
Hm. Maybe a more informative comment could be more clear. Something in the lines of

```suggestion
// # SAFETY
// As Gpiox uses dyn GpioExt internally, `Send` is not auto-implement. 
// But since GpioExt does only do atomic operations without side-effects we can assume, 
// that it safe to `Send` this type. 
```